os/drivers/testcase/testcase_drv.c

```diff
@@ -72,8 +73,142 @@ static int testdrv_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	 *   ioctl argument:  An integer value indicating the particular test to be run
 	 */
 
-	case TESTIOC_DRIVER_ANALOG: {
+	case TESTIOC_ANALOG: {
+
+	}
+	break;
+
+	case TESTIOC_CLOCK_ABSTIME2TICKS: {
+		base_time.tv_sec = (time_t)arg;
+		if (clock_abstime2ticks(CLOCK_REALTIME, &base_time, &ret) != OK) {
```

Comment:
1. header for clock_abstime2ticks API.
2. header for CLOCK_REALTIME

Reply:
I added that.